content/articles/launching-proppy.md

```diff
@@ -0,0 +1,35 @@
+Title: We are Wizards launch a new product
+Date: 2016-03-25
+Short_summary: After wrangling with proposals for a while we productized one of our internal tools.
+Category: Product
+Authors: Vincent, Tom
+
+*Reading time: ~3 minutes.*
+
+At We Are Wizards we send out a fair number of proposals. In our early
```

Comment:
Hm, I'd prefer to keep the first sentence out then. It appreciate what it is trying to add content-wise but I can't squeeze it in.